cleric-quests.lic

```diff
@@ -129,7 +129,11 @@ class ClericQuests
     bput('meditate', 'You close your eyes and begin to meditate')
     waitfor('once again changing and reforming into your Human form')
 
-    bput('dive deeper water', 'You take a deep breath')
+    case bput('dive deeper water', 'You take a deep breath', 'You get a sense that you have completed this')
+    when 'You get a sense that you have completed this'
+      pause 300
```

Comment:
Note: I guesstimated on this. I waited 4 minutes and I could complete it though, so made it 5 to be on the safe side.